airflow/ui/src/components/DurationChart.tsx

```diff
@@ -91,26 +94,51 @@ export const RunDuration = ({
   return (
     <Box>
       <Heading pb={2} size="sm" textAlign="center">
-        Last {totalEntries} runs
+        Last {pluralize(kind, entries.length)}
       </Heading>
       <Bar
         data={{
           datasets: [
             {
               backgroundColor: system.tokens.categoryMap.get("colors")?.get("queued.600")?.value as string,
-              data: runs.map((run: DAGRunResponse) => Number(getDuration(run.queued_at, run.start_date))),
+              data: entries.map((entry: RunResponse) => {
+                switch (kind) {
+                  case "Dag Run": {
+                    const obj = entry as DAGRunResponse;
```

Comment:
```suggestion
                    const run = entry as DAGRunResponse;
```

Let's have a more descriptive name.

Reply:
Thanks, done and rebased with main.